src/test/java/webdriverFramework/SSMission.java

```diff
@@ -1,35 +1,62 @@
 package webdriverFramework;
 
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import com.google.common.base.Function;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.remote.server.handler.FindElement;
-import org.openqa.selenium.support.ui.ExpectedCondition;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.Select;
-import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.ui.*;
 
 import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.fail;
 
-/**
- * Created by roman.pipchenko on 1/20/2017.
- */
 public class SSMission {
-    public static void main(String[] args) {
 
-        File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
-        if (file.exists()) {
-            System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
-        }
+    static WebDriver driver;
+    
+    public static String browser = "chrome";
+
+
+    @Rule
+    public TestRule screenshotTaker = new ScreenshotTaker((TakesScreenshot) driver);
 
-        System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\geckodriver.exe");
-        WebDriver driver = new FirefoxDriver();
+    @BeforeClass
+    public static void setup() {
+        if (browser.equals("chrome")) {
 
+            File file = new File("C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe");
+            if (file.exists()) {
+                System.setProperty("webdriver.chrome.driver", "src\\test\\resources\\chromedriver.exe");
+                driver = new ChromeDriver();
+            }
+        } else {
+            File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
+            if (file.exists()) {
+                System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
+            }
+            System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\geckodriver.exe");
+            driver = new FirefoxDriver();
+        }
+    }
+
+    @Test
+    public void testSS() throws InterruptedException {
 
         driver.get("https://www.ss.lv");
+        driver.manage().window().maximize();
+        assertTrue(driver.findElement(By.xpath("//*[@href='/ru/']")).isDisplayed());
```

Comment:
all locators extract to constants to avoid code duplication and make them reusable